plugin/mmdb/README.md

```diff
@@ -0,0 +1,56 @@
+# mmdb
+## Name
+*mmdb* - Lookup maxmind geoip2 databases then add associated data to the context request.
+
+## Description
+*mmdb* - Allows you to configure [geoIP2 maxmind databases](https://dev.maxmind.com/geoip/docs/databases) and automatically associate and DNS request data matching the client IP.
```

Comment:
```suggestion
The *mmdb* plugin allows configuring a  [geoIP2 maxmind databases](https://dev.maxmind.com/geoip/docs/databases) to automatically associate a geographic region by matching the client IP.
```

Reply:
Many of those databases provides useful data which is not geographic, see: https://dev.maxmind.com/geoip/docs/databases I feel with little effort (by handling their specific schemas) this plugin could support them all, allowing users to hook any of those databases instead of just doing geo with City, Country and Enterprise databases.

What do you think?

Reply:
Narrowed down to only one type.